com.unity.netcode.gameobjects/Tests/Editor/NetworkManagerConfigurationTests.cs

```diff
@@ -34,5 +34,41 @@ public void NestedNetworkManagerCheck()
             // Clean up
             Object.DestroyImmediate(parent);
         }
+
+        public enum NetworkObjectPlacement
+        {
+            Root,   // Added to the same root GameObject
+            Child   // Added to a child GameObject
+        }
+
+        [Test]
+        public void NetworkObjectNotAllowed([Values] NetworkObjectPlacement networkObjectPlacement)
```

Comment:
Nit, but I'd do "TestDisallowNetworkObjectsUnderNetworkManagers"

Reply:
This is the method:  NetworkManagerConfigurationTests.NetworkObjectNotAllowed
Do we really want to add another NetworkManager reference to this?
It would then be:
NetworkManagerConfigurationTests.TestDisallowNetworkObjectsUnderNetworkManagers
That seems a bit redundant?

Reply:
Ok, I'm sold